pipelines/visitQAEllipPlots.yaml

```diff
@@ -201,6 +201,31 @@ tasks:
         from lsst.analysis.drp.dataSelectors import (VisitPlotFlagSelector, SnSelector)
         from lsst.analysis.drp.calcFunctors import CalcE2
 
+  plot_ellipResids_quiver_visit:
+    # Make a quiver/whisker plot of (star-PSF) ellipticity residuals
+    class: lsst.analysis.drp.quiverPlotVisit.QuiverPlotVisitTask
+    config:
+      axisActions.zAction: SubtractColumns
+      axisActions.zAction.colA: CalcE
+      axisActions.zAction.colA.colXx: "ixx"
+      axisActions.zAction.colA.colYy: "iyy"
+      axisActions.zAction.colA.colXy: "ixy"
+      axisActions.zAction.colB: CalcE
+      axisActions.zAction.colB.colXx: "ixxPSF"
+      axisActions.zAction.colB.colYy: "iyyPSF"
+      axisActions.zAction.colB.colXy: "ixyPSF"
+      connections.plotName: Ediff_ixxPSF_quiver
+      selectorActions.flagSelector: VisitPlotFlagSelector
+      selectorActions.SnSelector: SnSelector
+      selectorActions.SnSelector.bands: [""]
+      selectorActions.SnSelector.fluxType: "psfFlux"
+      selectorActions.SnSelector.threshold: 10
+      axisLabels: {"x": "R.A. (Degrees)", "y": "Dec. (Degrees)", "z": "E Difference (eMEAS - ePSF)"}
+      python: |
+        from lsst.analysis.drp.dataSelectors import VisitPlotFlagSelector, GalaxyIdentifier, SnSelector
```

Comment:
You don't use `GalaxyIdentifier`--do you mean to be selecting only galaxies?

Reply:
I should remove it. I never need to select on galaxies to make these plots.